.gitignore

```diff
@@ -33,3 +33,5 @@ yarn-debug.log*
 
 /app/assets/builds/*
 !/app/assets/builds/.keep
+
+**/.DS_Store
```

Comment:
**suggestion:**, can you remove this from this PR and put it in your global `.gitignore` instead? Here are some directions https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer